### DISCUSSION TITLE
登幽州台歌@陈子昂·唐

### DISCUSSION THREAD

**Discussion by aspoem-giscus** [meetqy/aspoem#8] [Announcements]
# 登幽州台歌@陈子昂·唐

前不见古人，后不见来者。
念天地之悠悠，独怆然而泪下。 

https://aspoem.com/poem/132

**Comment by heloook** [comment]
既有时间又有空间，从古至今，最大气最有悟性的作品。

**Comment by Jessie-222** [comment]
非常实用的一个网站，超级nice~
